ui/core-react/src/core-react/listbox/Listbox.tsx

```diff
@@ -113,6 +113,7 @@ function processKeyboardNavigation(optionValues: ListboxItemProps[], itemIndex:
  * @alpha
  */
 export function Listbox(props: ListboxProps) {
+  // eslint-disable-next-line deprecation/deprecation
```

Comment:
Once you merge with master this deprecation has been updated and disabling eslint is not necessary.